pkgs/applications/misc/octoprint/plugins.nix

```diff
@@ -168,13 +168,13 @@ let
 
     printtimegenius = buildPlugin rec {
       pname = "PrintTimeGenius";
-      version = "2.0.2";
+      version = "2.2.1";
```

Comment:
In case you didn't notice: typically you also need to update the `marlin-calc` package to the version listed here: https://github.com/eyal0/OctoPrint-PrintTimeGenius/tree/master/octoprint_PrintTimeGenius/analyzers

This time there have been no changes to `marlin-calc` (except for adding a windows cross build target, which doesn't concern us at all), so you're fine.

Reply:
Oh ok, thanks for the heads up.